src/client/pages/__snapshots__/characters.spec.js.snap

```diff
@@ -5,5 +5,6 @@ exports[`Characters page component will render the Characters page 1`] = `
   <h2>
     Characters
   </h2>
+  <ConnectFunction />
```

Comment:
This is because react-redux v7 is using hooks, so [Enzyme isn't correctly picking up the display name.](https://github.com/airbnb/enzyme/issues/2107) Will keep an eye out for Enzyme updates to sync up with the latest react-redux.